src/ui/alert.js

```diff
@@ -0,0 +1,16 @@
+import swal from 'sweetalert';
+import DOM from './dom/dom';
+
+const defaultAlertOptions = {
+  closeOnClickOutside: false,
+  closeOnEsc: false
+};
+
+export default class Notify {
```

Comment:
nit: Can we export a function called alert rather than wrap it in a class? I also like to match the file name with the export for default exports, so exporting a single function named 'alert' would make the default export match the filename

Reply:
updated!